lib/evaluateFunctions.js

```diff
@@ -65,6 +68,229 @@ function evaluateFunctions(node) {
   return NodeStatus.noChange(node);
 }
 
+// Evaluate nthRoot() function
+// Returns a NodeStatus object.
+function nthRoot(node) {
+  if (!NodeType.isFunction(node, 'nthRoot')) {
+    return NodeStatus.noChange(node);
+  }
+
+  let oldNode = node;
+  let newNode = clone(node, false);
+  const radicandNode = newNode.args[0];
+  const rootNode = newNode.args.length === 2 ? newNode.args[1] : NodeCreator.constant(2);
+
+  if (NodeType.isOperator(radicandNode)) {
+    // if is operator and operation is addition, we can't do anything
+    if (radicandNode.op === '+' || radicandNode.op === '-') {
+      return NodeStatus.noChange(node);
+    }
+    else if (radicandNode.op === '*' || radicandNode.op === '/') {
+      // distribute the root into the children
```

Comment:
I'm not sure why this is a step - some tests with substep examples and examples of math in the code comments would both help